.github/ISSUE_TEMPLATE/config.yaml

```
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 📖 OpenFGA's Documentation
+    url: https://openfga.dev/docs
+    about: Check OpenFGA's documentation for an in-depth overview
+  - name: 👽 Discord
+    url: https://discord.gg/8naAwJfWN6
+    about: Join OpenFGA's active Discord community
```

Should we move this to slack/comunity page now?

```suggestion
  - name: 🏙️  Community
    url: https://openfga.dev/docs/community
    about: Join OpenFGA's active community
```